src/functions-and-arrays.js

```diff
@@ -1,73 +1,480 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(numOne, numTwo) {
+  if (numOne > numTwo) {
+    return numOne;
+  } else if (numTwo > numOne) {
+    return numTwo;
+  } else {
+    return numOne;
+  }
+}
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(arr) {
+  if (arr.length == 0) {
+    return null;
+  } else if (arr.length == 1) {
+    return arr[0];
+  }
+
+  var longestWord = "";
+  for (var i = 0; i < arr.length; i++) {
+    if (longestWord.length < arr[i].length) {
+      longestWord = arr[i];
+    }
+  }
+  return longestWord;
+}
 
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(nums) {
+  if (nums.length == 0) {
+    return 0;
+  } else if (nums.length == 1) {
+    return nums[0];
+  }
+  var sum = 0;
+  for (var i = 0; i < nums.length; i++) {
+    sum += nums[i];
+  }
+  if (sum === 0) {
```

Comment:
This conditional is highly redundant. You can see the logic says: if sum equals 0, return 0, else return sum. But in the case sum is 0 i could also return sum which is also 0 ;) It all works, of course, but you can easily reduce your code removing the conditional and just do: return sum.